ojd_daps_skills/utils/plotting.py

```diff
@@ -8,10 +8,6 @@
 
 ChartType = alt.vegalite.v4.api.Chart
 
-# Fonts and colours
-FONT = "Averta"  # should be changed depending on what your font is called in your system's font book
```

Comment:
can you believe I found out how everyone got averta font saved to their fontbook - there's a shared folder called 'Nesta new Visual Identity Sept 2021' that has the Averta font that saves to 'Averta' - who knew...!

Reply:
although I still think we should pass this as a parameter as people can save the name differently i.e. like you and i did